keepassxc-browser/content/fields.js

```diff
@@ -149,7 +149,8 @@ kpxcFields.getCombination = async function(field, givenType) {
         if (!givenType && Object.values(combination).find(c => c === field)) {
             return combination;
         } else if (givenType && combination[givenType]) {
-            if (combination[givenType] === field || combination[givenType].includes(field)) {
+            if (combination[givenType] === field
+                || (Array.isArray(combination[givenType]) && combination[givenType].includes(field))) {
                 return combination;
             }
         }
```

Comment:
This might be more readable like this:

```
if (givenType) {
    // strictly search a given type
	c = combination[givenType];
	if (c && (c === field || (Array.isArray(c) && c.includes(field)) {
		return combination;
	}
} else if (Object.values(combination).find(c => c === field)) {
	return combination;
}
```